Toluene/MittalSung_2007_RCM_Tc_1022K.yaml

```diff
@@ -0,0 +1,160 @@
+---
+file-authors:
+    - name: Morgan Mayer
+      ORCID: 0000-0001-7137-5721
+file-version: 0
+chemked-version: 0.4.1
+reference:
+    doi: 10.1016/j.combustflame.2007.04.014
+    authors:
+        - name: Gaurav Mittal
+        - name: Chih-Jen Sung
+    journal: Combustion and Flame
+    year: 2007
+    volume: 150
+    pages: 355-368
+    detail: Toluene, Tc=1022K, phi=1
+experiment-type: ignition delay
+apparatus:
+    kind: rapid compression machine
+    institution: Case Western Reserve University
+    facility: CWRU RCM
+datapoints:
+    - temperature:
+        - 297 kelvin
+      ignition-delay:
```

Comment:
Is there any uncertainty information available for these data?

Reply:
It looks like there is uncertainty information given. The authors give an initial temperature uncertainty of +/- 0.5 K. Is the initial temperature the temperature specified in the datapoints section? They specify a pressure transducer The authors also state an overall uncertainty of ignition delay is in a range of 7-11%. Would I add a relative uncertainty of 11% to the ignition delay field?

Reply:
Yes, there is an absolute uncertainty in initial temperature of 0.5 K I will add here. Also, the authors state that ignition delay measurements have an overall uncertainty in the range of 7.5-11%. Should I just add that there is a relative uncertainty of 11% for ignition delay?

Reply:
Yeah, probably best to assume worst-case if the authors aren't more specific.